widget/edit.html

```diff
@@ -77,9 +83,11 @@ <h6>Converts a given JSON input to an actionable Grid visualization</h6>
                 </div>
                 <div class="row">
                     <div class="col-md-8">
-                        <ul id="action-editor-without-record" class="list-group margin-top-md clearfix margin-bottom-0" data-as-sortable
-                            data-ng-model="config.selectedPlaybooksWithoutRecord" data-ng-show="playbookData">
-                            <li id="without-record-draggable-item-all{{$index}}" class="column-editor-block margin-bottom-sm"
+                        <ul id="action-editor-without-record" class="list-group margin-top-md clearfix margin-bottom-0"
+                            data-as-sortable data-ng-model="config.selectedPlaybooksWithoutRecord"
+                            data-ng-show="playbookData">
+                            <li id="without-record-draggable-item-all{{$index}}"
+                                class="column-editor-block margin-bottom-sm"
                                 data-ng-repeat="action in config.selectedPlaybooksWithoutRecord" data-as-sortable-item>
                                 <div cs-icons data-ng-model="action.icon" style="width: 50px; display: inline-block;">
```

Comment:
Avoid using styles, please use predefined classes for same
e.g width-50 and display-inline-block

Reply:
Done